packages/prompts/test/note.test.ts

```diff
@@ -63,4 +63,33 @@ describe.each(['true', 'false'])('note (isCI = %s)', (isCI) => {
 
 		expect(output.buffer).toMatchSnapshot();
 	});
+
+	test('don\'t overflow', () => {
+	  const lorem = [
+	    "Eiusmod sint officia proident qui ex tempor laborum ut. Voluptate dolore excepteur sunt adipisicing laborum non ea elit. Consectetur exercitation ad laboris consectetur nisi minim labore culpa enim. Eiusmod duis minim velit mollit nisi. Laboris magna qui fugiat sit minim. Aliqua id aute sunt proident do tempor mollit nulla veniam exercitation est ipsum occaecat nostrud. Exercitation ullamco proident ex ut et in ut.",
```

Comment:
just to simplify the test case, we could probably do something like this:

```ts
const input = `${'test string '.repeat(32)}\n`.repeat(4).trim();
```

this will just give us 4 lines of `test string` repeated a bunch